Sources/Sentry/SentryDataCategoryMapper.m

```diff
@@ -11,6 +11,7 @@
 NSString *const kSentryDataCategoryNameAttachment = @"attachment";
 NSString *const kSentryDataCategoryNameUserFeedback = @"user_report";
 NSString *const kSentryDataCategoryNameProfile = @"profile";
+NSString *const kSentryDataCategoryNameProfileChunk = @"profile_chunk";
```

Comment:
Didn't know that data category exists already 💯 
https://github.com/getsentry/relay/blob/23a48f2c7142ab940d5e6e3d3289c087eba2fcf2/py/sentry_relay/consts.py#L30